### PR TITLE
[ADF-2567] delete a version

### PR DESCRIPTION
**Pull request by DenisVuyka**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

ADF-2567: Delete a version
ADF-2632: Version Manager should support application configuration settings
ADF-2634: Node Version List actions are not localised

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: demo-shell/src/app.config.json [hunk 1]
```diff
@@ -484,5 +484,10 @@
                 "exif:exif": "*"
             }
         }
+    },
+    "adf-version-manager": {
```
Comment (eromano) [inline demo-shell/src/app.config.json]: The schema json need do be updated as well

File: demo-shell/src/app/components/files/files.component.html [hunk 2]
```diff
@@ -362,7 +362,8 @@
                         <adf-version-manager
                             [node]="documentList.selection[0].entry"
                             [showComments]="showVersionComments"
```
Comment (eromano) [inline demo-shell/src/app/components/files/files.component.html]: Why one property is enable and another allow as prefix? Can we define a standard?

Comment (DenisVuyka) [reply demo-shell/src/app/components/files/files.component.html]: What we use more and what you suggest? "enable" or "allow"? :) I am fine with anything

File: demo-shell/src/app.config.json (hunk 1 above)
Comment (DenisVuyka) [reply demo-shell/src/app.config.json]: added schema update

File: demo-shell/src/app/components/files/files.component.html (hunk 2 above)
Comment (DenisVuyka) [reply demo-shell/src/app/components/files/files.component.html]: changed to allow, as there are more "allow" usages for this feature and only 1 "enable"

**Comment by DenisVuyka** [comment]
added unit tests

File: demo-shell/src/app/components/files/files.component.html (hunk 2 above)
Comment (eromano) [reply demo-shell/src/app/components/files/files.component.html]: Good , I think we need to have a bigger discussion around all the property prefix for 3.0

**Pull request merged**